bundles/binding/org.openhab.binding.harmonyhub/src/main/java/org/openhab/binding/harmonyhub/HarmonyHubBindingProvider.java

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright (c) 2010-2015, openHAB.org and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.harmonyhub;
+
+import org.openhab.binding.harmonyhub.internal.HarmonyHubBindingConfig;
+import org.openhab.core.binding.BindingProvider;
+import org.openhab.core.items.Item;
+
+/**
+ * @author Dan Cunningham
+ * @since 1.7.0
+ */
+public interface HarmonyHubBindingProvider extends BindingProvider {
+    /**
+	 * Returns the name of the item
+	 * @param itemName
+	 * @return
+	 */
+	public Item getItem(String itemName);
```

Comment:
dealing with Items should be avoided wherever possible. Better work with the ItemName when possible